core/src/banking_stage/consumer.rs

```diff
@@ -1665,9 +1666,9 @@ mod tests {
             assert_eq!(
                 transaction_counts,
                 ExecuteAndCommitTransactionsCounts {
```

Comment:
should this struct not be renamed too?

Reply:
Yeah sure, I renamed to `LeaderProcessedTransactionCounts`